go/protocol/keybase1/extras.go

```diff
@@ -1892,11 +1892,18 @@ func (n ImplicitTeamDisplayName) String() string {
 	return name
 }
 
+const (
+	// LockIDVersion0 is the first ever version for lock ID format.
+	LockIDVersion0 byte = iota
+)
+
 // LockIDFromBytes takes the first 8 bytes of the sha512 over data, interprets
-// it as int64 using little endian, and returns the value as LockID.
+// it as int64 using big endian, and returns the value as LockID. First byte
+// is used as the version byte.
```

Comment:
"is used as" -> "is overwritten with"?  I was surprised the sha was reduced from 8 bytes to 7 when I read the code.